tests/suites/cloud_gce/create_storage_pool.sh

```diff
@@ -0,0 +1,35 @@
+run_create_storage_pool() {
+	echo
+
+	file="${TEST_DIR}/test-create-storage-pool.log"
+
+	ensure "test-create-storage-pool" "${file}"
+
+	juju create-storage-pool mygpd gce disk-type=pd-ssd
+	juju deploy postgresql --channel 14/stable --storage pgdata=20G,mygpd
+	wait_for_machine_agent_status "0" "started"
+
+	disk_name="$(juju status --format=json | jq -r '.storage.volumes["0"]."provider-id"')"
+	disk_info="$(gcloud compute disks list --filter="name=$disk_name" --format=json)"
+	test 20 -eq "$(jq -r '.[0].sizeGb' <<<"$disk_info")"
+	jq -r '.[0].type' <<<"$disk_info" | grep "/pd-ssd$"
+
+	# FIXME: temporarily use `--force` for `destroy-model`, as it may hang
```

Comment:
nit: use TODO
mention that the cause is a problem in detaching storage during model teardown